song/app.js

```diff
@@ -94,6 +98,54 @@ app.post("/content", async (req, res, next) => {
   );
   res.status(201).json({ message: "postCreated" });
 });
+// 좋아요 기능
+app.post("/like", async (req, res, next) => {
+  const { userId, postId } = req.body;
+  await appDataSource.query(
+    `
+    INSERT INTO likes(
+      user_id,
+      post_id
+    ) VALUES (?, ?)
+    `,
+    [userId, postId]
+  );
+
+  res.status(200).json({ message: "likeCreated" });
```

Comment:
```suggestion
  res.status(201).json({ message: "likeCreated" });
```
created라는 응답 메세지에 적합한 상태 코드를 쓰는 것이 적절해보입니다.